### ISSUE TITLE
Question about inner bezel

### ISSUE THREAD

**Issue by 8lall0** [Diepzeevogel/iMac-G3-replacement-parts#2]
Hi, i have a G3 with the inner bezel completely destroyed thanks to the worst packaging ever.
Can your inner bezel model replace it?

Thanks!

**Comment by undefik** [comment]
it's not the packaging's fault, mine was packaged in two or three layers of bubble wrap and cardboard, and the inner bezel was still shattered when I unpacked it.

Anyway, I would also like to know if this model is compatible with a stock CRT. I don't want a LCD in my G3.

**Comment by cfryerdev** [comment]
Yes, it works with the CRT... @MichalNemecek, this also happened to me, a 2000 iMac G3 DV 450... I had to add spacers for the bezel to fit the mounting screws as the designs above are too short, and a mount for the larger mic to fit into and mount. Note: I got lazy and just sanded the edges to match the curve for the mic mount though. 

Standoffs: https://github.com/cfryerdev/iMac-G3-replacement-parts/blob/master/Mounts/CRT/iMac_G3_Standoffs.stl

Mic Mount: https://github.com/cfryerdev/iMac-G3-replacement-parts/blob/master/Mounts/Mic/iMac_G3_Mic_Mount.stl

Adjustments I had to make: https://i.imgur.com/h6ASOyU.jpeg

**Comment by crozone** [comment]
I have an intact bezel and a 3D scanner, would a reference STL of the original part be useful?

**Comment by cfryerdev** [comment]
That would be awesome

**Comment by 8lall0** [comment]
> Yes, it works with the CRT... @MichalNemecek, this also happened to me, a 2000 iMac G3 DV 450... I had to add spacers for the bezel to fit the mounting screws as the designs above are too short, and a mount for the larger mic to fit into and mount. Note: I got lazy and just sanded the edges to match the curve for the mic mount though.
> 
> Standoffs: https://github.com/cfryerdev/iMac-G3-replacement-parts/blob/master/Mounts/CRT/iMac_G3_Standoffs.stl
> 
> Mic Mount: https://github.com/cfryerdev/iMac-G3-replacement-parts/blob/master/Mounts/Mic/iMac_G3_Mic_Mount.stl
> 
> Adjustments I had to make: https://i.imgur.com/h6ASOyU.jpeg

Do you have pictures on how did you mount the crt?

**Comment by CelvinBraun** [comment]
> I have an intact bezel and a 3D scanner, would a reference STL of the original part be useful?

@crozone - Did you ever had the chance to do the 3D scan?

**Comment by CelvinBraun** [comment]
> > Yes, it works with the CRT... @MichalNemecek, this also happened to me, a 2000 iMac G3 DV 450... I had to add spacers for the bezel to fit the mounting screws as the designs above are too short, and a mount for the larger mic to fit into and mount. Note: I got lazy and just sanded the edges to match the curve for the mic mount though.
> 
> > 
> 
> > Standoffs: https://github.com/cfryerdev/iMac-G3-replacement-parts/blob/master/Mounts/CRT/iMac_G3_Standoffs.stl
> 
> > 
> 
> > Mic Mount: https://github.com/cfryerdev/iMac-G3-replacement-parts/blob/master/Mounts/Mic/iMac_G3_Mic_Mount.stl
> 
> > 
> 
> > Adjustments I had to make: https://i.imgur.com/h6ASOyU.jpeg
> 
> 
> 
> Do you have pictures on how did you mount the crt?

@cfryerdev - I would be interested in this as well. And thanks for the additional parts you made available! 😃
